.devcontainer/startup.bash

```diff
@@ -7,14 +7,16 @@ docker compose up &> .devcontainer/output/docker_compose_up.out &
 
 sleep 30
 
-if [ ! -f .env ]; then 
-    cp .env.example .env 
-    sed -i "/^# PG/s/^# //g" .env 
-fi 
+Xvfb $DISPLAY -ac &> .devcontainer/output/Xvfb.out &
+
+if [ ! -f .env ]; then
+    cp .env.example .env
+    sed -i "/^# PG/s/^# //g" .env
```

Comment:
Oh damn `sed`

Reply:
Seemed like the easiest way to setup the env file for docker stuff. I'm just glad it works.